### PR TITLE
Update README: Clarify which standards are useful submissions

### PR DESCRIPTION
**Pull request by fulldecent**
Here is the blog post https://blog.phor.net/2022/09/30/What-kinds-of-things-should-be-standardized.html

This PR adds it as recommended reading.

### CODE REVIEW

**Review by Pandapip1 — APPROVED** [review]
+0. I don't like the fact that this is an external link, but it's also not part of an EIP so it's easier to remove if needbe.

**Comment by xinbenlv** [comment]
@fulldecent 


I like this personal essay, but I think publishing it at personal venue is better. As of now, I am not sure EIP-1 / README is a good place for personal essay. 

Until EIP-1, README  decide to be linking to personal essays  and is committed to have a complete and balanced representations on views on the same issue, I would not be in support of this proposed inclusion.

**Comment by SamWilsn** [comment]
This is good content, but I shy away from adding an external link. Would you be open to publishing this as an EIP-1 asset?

**Comment by fulldecent** [comment]
@SamWilsn Thank you. So adding into EIP-1 or a separate item?

**Comment by Pandapip1** [comment]
> So adding into EIP-1 or a separate item?

Put a Markdown version of the article in `/assets/eip-1/useful-submissions.md`.

**Comment by xinbenlv** [comment]
I object to add this doc to EIP repository as per my commented above

**Comment by fulldecent** [comment]
@Pandapip1 Updated presentation to the format as you said.

File: assets/eip-1/useful-submissions.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+But if you really WANT to make something and standardize it, here's how.
+
+1. Go make a thing that is useful
+2. Make it successful
+3. Make other people want to interoperate with your thing
+4. Make other people want to copy you
+5. Then go standardize it
+
+Remember, CryptoKitties and CryptoPunks came first, the blockchain NFT standard come afterwards. Don't think that having some standard will make your product successful, that causality is backwards.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:53]:
```suggestion
Remember, CryptoKitties and CryptoPunks came first, and [the blockchain NFT standard](../../EIPS/eip-721.md) come afterward. Having a standard will not make your product successful. Instead, having a project will make your standard successful.
```

File: assets/eip-1/useful-submissions.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+FYI, in the United States, the post office has their own police. Watch out for that.
+
+✅ If you routinely use postcard stamps to mail letters that are not postcard size you will go to jail. Multiple people mail postcards and depend on not going to jail... this deserves to be standardized.
+
+❌ Your smart contract uses a specific way to track inclusion of addresses in a set. Nobody cares or is affected by which specific way you track inclusion of addresses in a set. This does not deserve to be standardized.
+
+An easy, low-effort, low-quality way to find a bunch of things to standardize is to find a popular repository of smart contracts, and then write every one of them up with a short rationale. This is fun for practice, but please do not publish these as it will be a waste of other people's time to read. Nobody depends on these details.
+
+But if you really WANT to make something and standardize it, here's how.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:45]:
```suggestion
But if you really WANT to standardize something, here's how:
```
Make this a bit more succinct

File: assets/eip-1/useful-submissions.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+1. Go make a thing that is useful
+2. Make it successful
+3. Make other people want to interoperate with your thing
+4. Make other people want to copy you
+5. Then go standardize it
+
+Remember, CryptoKitties and CryptoPunks came first, the blockchain NFT standard come afterwards. Don't think that having some standard will make your product successful, that causality is backwards.
+
+William Entriken / 2022-09-30 / this article is cross-posted with [phor.net](https://blog.phor.net/2022/09/30/What-kinds-of-things-should-be-standardized.html).
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:55]:
```suggestion
William Entriken / 2022-09-30 / Derived from a [phor.net](https://blog.phor.net/2022/09/30/What-kinds-of-things-should-be-standardized.html) article
```

File: assets/eip-1/useful-submissions.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+<mark>A standard only deserves to be written if multiple people adhere to it and people depend on its surface area.</mark>
+
+FYI, in the United States, the post office has their own police. Watch out for that.
+
+✅ If you routinely use postcard stamps to mail letters that are not postcard size you will go to jail. Multiple people mail postcards and depend on not going to jail... this deserves to be standardized.
+
+❌ Your smart contract uses a specific way to track inclusion of addresses in a set. Nobody cares or is affected by which specific way you track inclusion of addresses in a set. This does not deserve to be standardized.
+
+An easy, low-effort, low-quality way to find a bunch of things to standardize is to find a popular repository of smart contracts, and then write every one of them up with a short rationale. This is fun for practice, but please do not publish these as it will be a waste of other people's time to read. Nobody depends on these details.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:43]:
```suggestion
An easy way to find a bunch of things to standardize is to find a popular repository of smart contracts, and then write every one of them up with a short rationale. This is fun as practice, but unless you have found a strong motivation, please do not submit these.
```
Although I generally agree with the sentiment, those last few sentences are too harsh, even by my standards.

File: assets/eip-1/useful-submissions.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+* When I walk down the street, I don't skip. That's **a walking standard**.
+* My smart contract allows to mint pictures of tables using a `mintTable` function. This is **a minting standard for NFTs**.
+* I wrote some software and created test cases. All of those test cases are **a software standard**.
+
+Some of these behaviors might be important. Maybe you don't like skipping, you can adopt that behavior. But as a standard these are useless, I hope you understand. But let's articulate why.
+
+<mark>A standard only deserves to be written if multiple people adhere to it and people depend on its surface area.</mark>
+
+FYI, in the United States, the post office has their own police. Watch out for that.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:37]:
```suggestion
(Note: In the United States, the post office has its own police. Watch out!)
```
Might also be helpful to put that after the ✅

File: assets/eip-1/useful-submissions.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+We have looked at a few example of things that have been standardized, different kinds of standards. And you might think, anything can be rewritten as a standard if you really wanted to. And you're right. Here are some contrived examples:
+
+* When I walk down the street, I don't skip. That's **a walking standard**.
+* My smart contract allows to mint pictures of tables using a `mintTable` function. This is **a minting standard for NFTs**.
+* I wrote some software and created test cases. All of those test cases are **a software standard**.
+
+Some of these behaviors might be important. Maybe you don't like skipping, you can adopt that behavior. But as a standard these are useless, I hope you understand. But let's articulate why.
+
+<mark>A standard only deserves to be written if multiple people adhere to it and people depend on its surface area.</mark>
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:35]:
```suggestion
**A standard only deserves to be written if multiple people adhere to it and people depend on its surface area.**
```

File: assets/eip-1/useful-submissions.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+# Useful submissions
+
+A <dfn>standard</dfn>, in technical settings, is an established specification which allows any compliant thing to work with any compliant user of that thing, also known as cross-connect. It is a set of restrictions, or marketing claims, incumbent upon the standardized product.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:3]:
```suggestion
A *standard*, in technical settings, is an established specification that allows any compliant thing to work with any compliant user of that thing, also known as cross-connect. It is a set of restrictions or marketing claims, incumbent upon the standardized product.
```

File: assets/eip-1/useful-submissions.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+We call the compliant thing a <dfn>producer</dfn>. And the thing that is relying on those specifications is the <dfn>consumer</dfn>.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:5]:
```suggestion
We call the compliant thing a *producer*. And the thing that is relying on those specifications is the *consumer*.
```

File: assets/eip-1/useful-submissions.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+And in technical circles, we call the specific parts that are standardized as its <dfn>surface area</dfn>.
```
Comment (Pandapip1) [inline assets/eip-1/useful-submissions.md:7]:
```suggestion
And in technical circles, we call the specific parts that are standardized as its *surface area*.
```
These short sentences should probably be all grouped into one paragraph.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Comment by xinbenlv** [comment]
Hi @Pandapip1

Based on this “suggestion”, ERC165 shouldnt be proposed. ERC5289 that @Pandapip1 you proposed should also not be proposed as an EIP because there were no pri~adoption. And many EIPs shall not be proposed

This is why oppose this "suggestion".

**Comment by Pandapip1** [comment]
Well, as it turns out, EIP-165 could have been improved had it been tried as a project beforehand. I think EIP-165 is a *good* example of why this should be the case.

**Comment by xinbenlv** [comment]
@Pandapip1 it might be the case. But there is also merit in publishing thr ERC first that drives consensus and adoption. 

I think the "suggestion" is to say that ERC165 or ERC5289 and ENS standards shall not be even worth proposed unless tried and adopted first. I think this is not the case for many of the existing and useful ERCs. 4337 is also a counter example

Another bad example is https://eips.ethereum.org/EIPS/eip-5313 which provides very limited value to the existing of ERC173

Regardless, I think inclusion of this "suggestion" I think is worthy of a EMF discussion thread.

**Pull request closed without merging**